### PR TITLE
Update Infinit configuration.

### PR DESCRIPTION
**Pull request by cadeuh**
Changed the parsed URLs and put some rankings on it.

### CODE REVIEW

File: infinit.json [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
+    {
+      "url": "https://infinit.sh/documentation/status",
+      "page_rank": -2
+    },
+    {
+      "url": "https://infinit.sh/documentation/changelog",
+      "page_rank": -2
+    },
+    {
+      "url": "https://infinit.sh/documentation/.*",
```
Comment (redox) [inline infinit.json]: I think you need to put that rule last; otherwise it will bypass the next one. Can you guys confirm @maxiloc @ElPicador ?

Comment (maxiloc) [inline infinit.json]: Yes you just need to switch the last 2. Because we use the page_rank of the first matching start_url

**Comment by maxiloc** [comment]
I updated the configuration manually, so I am closing this. Thanks for contributing

**Comment by cadeuh** [comment]
Thanks!

**Pull request closed without merging**